Extensions/Leaderboards/JsExtension.js

```diff
@@ -86,6 +91,48 @@ module.exports = {
       .setIncludeFile('Extensions/Leaderboards/leaderboardstools.js')
       .setFunctionName('gdjs.evtTools.leaderboards.hasLastEntrySaveFailed');
 
+    extension
+      .addCondition(
```

Comment:
Nitpicking: we took the convention to start all conditions and actions by a capital (PascalCase)